handout/architect/10_ways_to_misuse_angular.md

```diff
@@ -0,0 +1,103 @@
+# Ten Ways to Misuse Angular
+
+1. Ignoring a component’s host element
+    - No need to wrap templates in a `<div>`
+    - Use `:host` selector
+    - Use `@HostBinding` and `@HostListener`
+    - Subtle differences host metadata vs. `@Host*` decorators
+1. Avoiding observables
+    - Leverage observable operators
+    - Merge and transform data in streams rather than direct data mutation
+    - Don’t prematurely subscribe
+    - Leverage `async` pipe where appropriate
+1. Using the raw `Http` service
+    - Avoid using `Http` in components
+        - Use services instead
+        - Map out `res.json()` as part of a method call
+    - Work with data shapes that make sense for the application
+        - Don’t necessarily consume data as returned from a API
+        - Use a transformation layer to massage data into a shape that makes sense
+1. Not considering the injector tree when lazy loading modules
+    - DI may behave differently than expected
+    - Lazy loaded module providers will not be available until the module has been loaded in
+    - Lazy loaded module providers will not be globally available but rather scoped to the lazy module
+    - Prefer to provide at the root module level
+    - Leverage `forRoot` / `forChild` configuration
+1. Confusing the source of truth between router and application state
+    - State transitions should be links or router navigation calls
+    - Hydrate / mutate state based on router events
+        - Dispatch actions to store accordingly
+1. Excessive encapsulation of child components
```

Comment:
This can use rewording, but the point here is that content projection should be leveraged whereas often content projection is ignored which can lead to deep component hierarchies necessitating Input/Output chaining, and overly-specific components. The biggest and most obvious use case here is with forms

Perhaps a better name is "Ignoring content projection"?